.travis.yml

```diff
@@ -18,9 +20,12 @@ jobs:
             done
             (exit $CLIPPY_STATUS)
           fi
+    - name: "Unstable"
+      script:
+        - cargo test --all --all-features
     - name: "Benchmarks"
       script:
-        - (cd benchmarks && cargo test --benches)
+        - (cd benchmarks && cargo test --benches --features unstable)
```

Comment:
[Travis](https://travis-ci.org/github/lambda-fairy/maud/jobs/708836791) doesn't like this:

```
$ (cd benchmarks && cargo test --benches --features unstable)
    Updating crates.io index
error: Package `benchmarks v0.1.2 (/home/travis/build/lambda-fairy/maud/benchmarks)` does not have these features: `unstable`
The command "(cd benchmarks && cargo test --benches --features unstable)" exited with 101.
```

You might need to update `benchmarks/Cargo.toml` as well.

(There's also a [Clippy failure](https://travis-ci.org/github/lambda-fairy/maud/jobs/708836789), but that's not related to this PR so I'll fix that myself.)

Reply:
(I've fixed the Clippy issue in #203.)

Reply:
Cool, thanks! I've rebased the branch onto master, so that should resolve this!